v2/internal/pkg/clusterresources/clusterresources.go

```diff
@@ -130,6 +132,11 @@ func (o *ClusterResourcesGenerator) generateITMS(mirrorsByCategory []categorized
 			},
 			ObjectMeta: metav1.ObjectMeta{
 				Name: "itms-" + catMirrors.category.toString() + "-0",
+				Annotations: map[string]string{
+					"createdBy":         "oc-mirror --v2",
+					"createdAt":         time.Now().UTC().Format(time.RFC850),
+					"oc-mirror_version": version.Get().GitVersion,
```

Comment:
This is not going to return the proper git version atm because we're not setting it during build:
```fish
❯ ./bin/oc-mirror --v2 version

2025/03/11 10:27:23  [INFO]   : 👋 Hello, welcome to oc-mirror
2025/03/11 10:27:23  [INFO]   : ⚙️  setting up the environment for you...
WARNING: This version information is deprecated and will be replaced with the output from --short. Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"", Minor:"", GitVersion:"v0.0.0-unknown", GitCommit:"", GitTreeState:"", BuildDate:"", GoVersion:"go1.23.6", Compiler:"gc", Platform:"linux/amd64"}
```
I'll fix this in a separate bug.